build.gradle

```diff
@@ -1,16 +1,20 @@
 buildscript {
     ext {
+        minSdkVersion = 21
+        targetSdkVersion = 30
+        compileSdkVersion = 30
+
         kotlinVersion = '1.4.31'
         androidGradleVersion = '4.1.2'
-        coroutineVersion = '1.4.2'
+        coroutineVersion = '1.4.3'
 
         // Google libraries
         activityVersion = '1.2.1'
         appCompatVersion = '1.2.0'
         constraintLayoutVersion = '2.0.4'
         materialComponentsVersion = '1.3.0'
         roomVersion = '2.2.6'
-        fragmentVersion = '1.3.0'
+        fragmentVersion = '1.3.1'
```

Comment:
You need to revert also this 👍  The reason is explained here: https://github.com/ChuckerTeam/chucker/pull/577

Reply:
Done